frontend/src/components/hamburger.js

```diff
@@ -0,0 +1,3 @@
+export default function hamburgerComponent() {
+	console.log('*** I will be a hamburger component');
```

Comment:
This will be the future hamburger component that will apply & remove classes etc to make the hamburger work across different screen sizes